lib/routes/jisilu/people.ts

```diff
@@ -0,0 +1,103 @@
+import { type CheerioAPI, load } from 'cheerio';
+import { type Context } from 'hono';
+
+import { type DataItem, type Route, type Data, ViewType } from '@/types';
+
+import ofetch from '@/utils/ofetch';
+
+import { rootUrl, processItems } from './util';
+import InvalidParameterError from '@/errors/types/invalid-parameter';
+
+const actions: { [key: string]: string } = {
+    questions: '101',
+    answers: '201',
+};
+
+export const handler = async (ctx: Context): Promise<Data> => {
+    const { id, type = 'questions' } = ctx.req.param();
+
+    if (!id) {
+        throw new InvalidParameterError('请填入合法的用户 id，参见用户排名 https://www.jisilu.cn/users/');
+    }
```

Comment:
Unreachable condition. https://github.com/DIYgod/RSSHub/blob/87df3dc03815e7e3a2f43ec3c5acb5378298594e/lib/routes/jisilu/people.ts#L64 defines `id` as compulsory parameter. User who visiting the route without providing `id` will see a `NotFoundError` instead